vendor/brave-ios/Ledger/BATBraveLedger.h

```diff
@@ -132,9 +132,6 @@ NS_SWIFT_NAME(BraveLedger)
 
 - (void)listRecurringTips:(void (^)(NSArray<BATPublisherInfo *> *))completion;
 
-- (void)addRecurringTipToPublisherWithId:(NSString *)publisherId
```

Comment:
On second thought, please leave this here, we are going to use it anyways from UI to add recurring tip